### PR TITLE
refactor(quarks): FRMW-415 Status and FRMW-416 StatusUtil

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-415 (`rxStatus.Status`)
JIRA: https://jira.rax.io/browse/FRMW-416 (`rxStatus.StatusUtil`)

- [x] Dev LGTM
- [ ] Dev LGTM
- [x] Design LGTM

---
Refactor `rxStatus.Status` and `rxStatus.StatusUtil` to Quark Services.  This PR removes `rxStatus` component.

### CODE REVIEW

**Comment by 100stacks** [comment]
## `Status` Quark Service

### Demo Page
screen shot 2016-02-11 at 3 58 16 pm

### API Docs
screen shot 2016-02-11 at 4 03 57 pm

**Comment by 100stacks** [comment]
## `StatusUtil` Quark Service

### Demo Page
screen shot 2016-02-11 at 4 20 05 pm

### API Docs
screen shot 2016-02-11 at 4 19 17 pm

File: src/quarks/StatusUtil/scripts/StatusUtil.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
  * @description
- * Manipulates references to needed $scope input for proper notification functionality
+ * Manipulates required references to $scope input for proper notification functionality.
  *
  * @example
  * <pre>
  * $rootScope.$on('$routeChangeSuccess', function () {
  *     Status.setScope(); // no input results in $rootScope being used
- *     Status.setUtil($rootScope); // forcibly set $rootScope as the scope to be used
+ *     Status.setScope($rootScope); // forcibly set $rootScope as the scope to be used
```
Comment (CITguy) [inline src/quarks/StatusUtil/scripts/StatusUtil.js]: :+1: nice catch

**Comment by CITguy** [comment]
LGTM

**Comment by ssalinas24** [comment]
LGTM

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**